src/the_way/cli.rs

```diff
@@ -71,6 +71,17 @@ pub enum TheWayCLI {
         #[structopt(parse(from_os_str))]
         file: Option<PathBuf>,
     },
+    /// Imports code snippets from a Gist given the URL.
+    ///
+    /// Multiple files will be converted to separate snippets.
+    /// Snippet description is created based on Gist description and file name with the format 
+    /// "<gist_description> - <file_name>".
+    /// Each snippet will be tagged with "gist" and its Gist ID.
+    /// Works for both secret and public gists.
+    ImportGist {
```

Comment:
Actually I was thinking the `import` function could have a `-g`/`--gist` flag, but maybe this way is better, though it's a bit long to type. Another option is to modify the import function to take `-f` for file and `-g` for gist. What are your thoughts on the usability of this?

Reply:
Oh right we could just add a `-g <gist_url>` and keep the optional "file" arg as is.
E.g.:
```
the-way import -g https://gist.github.com/xiaochuanyu/e5deab8d78ce838f22f160c9b14daf17
```

Reply:
Yes, that looks good!